src/uitpas/search/detail/UiTPasAdvantageDescription.js

```diff
@@ -4,9 +4,14 @@ import { map } from 'lodash';
 import moment from 'moment';
 
 import { UiTImage } from '../component/UiTImage';
-import { UiTPasCounter } from './UiTPasCounter';
 import { LastChanceLabel } from '../component/LastChanceLabel';
 import { joinNicely } from '../helper/UiTPasArrayUtils';
+import UiTPasSearchConfig from '../UiTPasSearchConfig';
+
+const CashInType = {
+    ONLINE: 'ONLINE',
+    FYSICAL: 'FYSICAL',
```

Comment:
I guess there is no way to use the correct spelling (`physical`)?

Reply:
I removed it since it wasn't used